opensir/models/sirx.py

```diff
@@ -172,6 +173,36 @@ def set_initial_conds(self, array=None, n_S0=None, n_I0=None, n_R0=None, n_X0=No
         self.w0 = arr / self.pop
         return self
 
+    def plot(self):
+        if self.sol is None:
+            raise self.InitializationError(
+                "Model must be either solved or fitted before plotting"
+            )
+
+        t = self.fetch()[:, 0]
+        n_i = self.fetch()[:, 2]
+        n_x = self.fetch()[:, 4]
+
+        fig, ax1 = plt.subplots(figsize=[5, 5])
+
+        color = "tab:red"
+        ax1.set_xlabel("Time / days", size=14)
+        ax1.set_ylabel("Number of infected $N_I$", size=14)
+        ax1.plot(t, n_i, color=color)
+        ax1.tick_params(axis="y", labelcolor=color)
+        ax1.tick_params(axis="both", labelsize=13)
+
+        ax2 = ax1.twinx()  # instantiate a second axes that shares the x coordinate
+
+        color = "tab:blue"
+        ax2.set_ylabel("Number of quarantined $N_X$", size=14)
+        ax2.plot(t, n_x, color=color, linestyle=":", linewidth=4)
+        ax2.tick_params(axis="y", labelcolor=color)
+        ax2.tick_params(axis="both", labelsize=13)
+
+        plt.title("SIR-X model predictions", size=16)
+        plt.show()
```

Comment:
I see there is a lot of repetition between this model and the SIR model. Maybe (not in this PR, but in a future one) we could refactor this to DRY out the code. Maybe use inheritance, or some other pattern.

Reply:
Let's implement this in another PR for plot, which will also deal with exporting a svg and the cli, following your issue #81